### PR TITLE
gui: Add the IsKeyPressed control to the buttons

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- gui: Add the IsKeyPressed control to the buttons
- gui/live area: Correct the description in live area help

thx Mr. @Zangetsu38 help me 🙏.

### CODE REVIEW

File: lang/ja.xml [hunk 1]
```diff
@@ -430,7 +430,6 @@ Vita3Kをお楽しみください。</completed_setup>
 			<browse_page>ページの閲覧</browse_page>
 			<browse_page_control>方向キー、左ステック、ホイールの上下またはスライダーまたは&lt;/&gt;</browse_page_control>
 			<hide_show>ボタンの表示/非表示</hide_show>
-			<hide_show_control>右クリック</hide_show_control>
```
Comment (Croden1999) [inline lang/ja.xml:433]: @nishinji need ur help

File: lang/ru.xml [hunk 2]
```diff
@@ -448,7 +448,6 @@
 			<browse_page_control>D-pad, л. стик влево/вправо, колёсико вверх/вниз 
 либо нажав &lt;/&gt;</browse_page_control>
 			<hide_show>Скрыть/показать кнопки</hide_show>
-			<hide_show_control>Правая кнопка мышки</hide_show_control>
```
Comment (Croden1999) [inline lang/ru.xml:451]: @bookmist , @KaneDbD  need ur help

File: lang/ja.xml (hunk 1 above)
Comment (nishinji) [reply lang/ja.xml:433]: "左クリックまたは△ボタンを押す"

Comment (Croden1999) [reply lang/ja.xml:433]: done

File: vita3k/gui/src/manual.cpp [hunk 3]
```diff
@@ -119,12 +123,9 @@ bool init_manual(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path
             }
         }
     }
-
```
Comment (Zangetsu38) [inline vita3k/gui/src/manual.cpp:122]: Keep new line here 👌

File: vita3k/interface.cpp [hunk 4]
```diff
@@ -637,6 +637,8 @@ bool handle_events(EmuEnvState &emuenv, GuiState &gui) {
                     return SCE_CTRL_CIRCLE;
                 else if (scancode == emuenv.cfg.keyboard_button_cross)
                     return SCE_CTRL_CROSS;
+                else if (scancode == emuenv.cfg.keyboard_button_triangle)
```
Comment (Zangetsu38) [inline vita3k/interface.cpp:640]: Move this after r1, line 636

Comment (Croden1999) [reply vita3k/interface.cpp]: done

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**